modules/storage-persistent-storage-pv.adoc

```diff
@@ -250,13 +250,13 @@ apiVersion: v1
 kind: PersistentVolume
 metadata:
   name: pv0001
-  annotations:
-    volume.beta.kubernetes.io/mount-options: rw,nfsvers=4,noexec <1>
 spec:
   capacity:
     storage: 1Gi
   accessModes:
   - ReadWriteOnce
+  mountOptions: <1>
+  - nfsvers=4.1
```

Comment:
Add two spaces to have the option one level inside of mount options, adding two spaces before "- nfsvers=4.1"

Reply:
Done! PTAL and let me know your feedback.